pkg/crc/machine/machine.go

```diff
@@ -107,6 +107,16 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error getting bundle metadata: %v", err)
 		}
 
+		// Check if certificate is going to expire in next 7 days
+		buildTime, err := crcBundleMetadata.GetBundleBuildTime()
+		if err != nil {
+			result.Error = err.Error()
+			return *result, errors.Newf("Error getting bundle build time: %v", err)
+		}
+		if cluster.CheckCertsValidityUsingBundleBuildTime(buildTime) {
+			logging.Warn("Bundle certificates are going to expire in 7 days, better to use new release")
```

Comment:
Better to say soon, instead of 7 days, as this should start a countdown otherwise.. even on the 6th day it'll say will expire in 7 days.

Reply:
@anjannath Good catch, I added the counter now which will tell within 7 days, how many days left for cert expiry.